plugin_repackaging.sh

```
@@ -12,183 +12,155 @@ PIP_MIRROR_URL="${PIP_MIRROR_URL:-$DEFAULT_PIP_MIRROR_URL}"
-	;;
-	'github')
-	github $@
-	;;
-	'local')
-	_local $@
-	;;
-	*)
-
-print_usage
-exit 1
+    market) market $@ ;;
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)